src/workerd/io/worker-interface.capnp

```diff
@@ -244,8 +233,7 @@ struct Trace @0x8e8d911203762d34 {
       email @5 :EmailEventInfo;
       trace @6 :TraceEventInfo;
       hibernatableWebSocket @7 :HibernatableWebSocketEventInfo;
-      resume @8 :Resume;
-      custom @9 :CustomEventInfo;
+      custom @8 :CustomEventInfo;
```

Comment:
As discussed previously, Onset and TailEvent below are APIs that we do not make any stability guarantees for yet, so changing them is acceptable.